TsRandomizer/Randomisation/ItemLocationMap.cs

```diff
@@ -206,12 +206,12 @@ static int CalculateCapacity(SeedOptions options)
 		void AddPresentItemLocations()
 		{
 			areaName = "Tutorial";
-			Add(ItemKey.TutorialMeleeOrb, "Yo Momma", ItemProvider.Get(EInventoryOrbType.Blue, EOrbSlot.Melee));
-			Add(ItemKey.TutorialSpellOrb, "Yo Momma", ItemProvider.Get(EInventoryOrbType.Blue, EOrbSlot.Spell));
+			Add(ItemKey.TutorialMeleeOrb, "Mommy's Orbs", ItemProvider.Get(EInventoryOrbType.Blue, EOrbSlot.Melee)); //more descriptive while retaining the joke
+			Add(ItemKey.TutorialSpellOrb, "Mommy's Necklace", ItemProvider.Get(EInventoryOrbType.Blue, EOrbSlot.Spell)); //see above
 			areaName = "Lake Desolation";
-			Add(new ItemKey(1, 1, 1528, 144), "Starter chest 2", ItemProvider.Get(EInventoryUseItemType.FuturePotion), AccessToLakeDesolation);
-			Add(new ItemKey(1, 15, 264, 144), "Starter chest 3", ItemProvider.Get(EInventoryEquipmentType.OldCoat), AccessToLakeDesolation);
-			Add(new ItemKey(1, 25, 296, 176), "Starter chest 1", ItemProvider.Get(EInventoryUseItemType.FutureHiPotion), AccessToLakeDesolation);
+			Add(new ItemKey(1, 1, 1528, 144), "We are not in Kansas anymore", ItemProvider.Get(EInventoryUseItemType.FuturePotion), AccessToLakeDesolation); //these aren't always the Starter Chests
+			Add(new ItemKey(1, 15, 264, 144), "Old Coat chest", ItemProvider.Get(EInventoryEquipmentType.OldCoat), AccessToLakeDesolation); //see above
+			Add(new ItemKey(1, 25, 296, 176), "Pyramid keys room", ItemProvider.Get(EInventoryUseItemType.FutureHiPotion), AccessToLakeDesolation); //see "see above" above
```

Comment:
Why is this Pyramid keys room and the other one Pyramid keys chest? They both have chests in them so how are you supposed to tell them apart?